trunk/conf/full.conf

```diff
@@ -550,6 +550,11 @@ vhost rtc.vhost.srs.com {
         # Overwrite by env SRS_VHOST_RTC_KEEP_BFRAME for all vhosts.
         # default: off
         keep_bframe off;
+        # The transcode audio bitrate, for RTMP to RTC.
+        # Overwrite by env SRS_VHOST_RTC_OPUS_BITRATE for all vhosts.
+        # [8000, 320000]
```

Comment:
Here, all possible values should be given, otherwise, it is not known how to choose, and you need to search for available values.